frontend/src/app/components/address-labels/address-labels.component.html

```diff
@@ -1,5 +1,4 @@
 <span
   *ngIf="label"
   class="badge badge-pill badge-warning"
-  i18n="address-labels.multisig"
```

Comment:
had to removed in b5ef148, there is no need for localization with just a variable `label` and no text before of after it